content/02.introduction.md

```diff
@@ -1,10 +1,14 @@
 ## Introduction
 
-- Importance of computation in biological research and scope.
-- Importance to reproducibility in computational biology.
-- Mentioning previous literature and some of their major takeaways and topics not fully covered in here (what kind of languages to learn, advanced text editors/IDEs,
-- The computational biology best-practices continuum: reproducibility at the personal level, sharing our research with others and over time.
-- Addressing the audience: computational biologists researchers – data analysts, workflow designers, software developers, mathematical modelers, etc.
-- Structure of this manuscript: mention Figure 1 and Figure 2.
-  - **Figure 1:** Funnel structure for better computational biology.
-  - **Figure 2:** Types of computational biology projects per level.
+Since Margaret Dayhoff pioneered the field of bioinformatics back in the decade of the sixties, the application of computational tools to the field of biology has vastly grown in scope and impact. Nowadays, biotechnological and biomedical research are routinely fed by the insights arising from novel computational approaches, machine learning algorithms and mathematical models. The ever increasing amount of biological data and the exponential growth in computing power will amplify this trend in the years to come.
+
+The use of computers to address biological matters encompasses a wide array of applications usually grouped under the terms of computational biology and bioinformatics. Although distinct definitions have been delineated for each one, here we will consider “computational biology” as the umbrella term for any application that involves the intersection of computing and modern biological data. As such, a computational biologist can be a bioinformatician, a statistician, a mathematical modeler, a data analyst, a data engineer, a software developer, and many others. In praxis, the modern computational biologist will be a “scientist of many hats”, taking on several of the duties listed above. But first and foremost, we will consider a computational biologist as a scientist whose ultimate goal is answering a biological question or addressing a need in the life sciences.
+
+Computing requires following specific practices to enable shareable, reproducible and sustainable outputs that stand the test of time. Computing-heavy disciplines such as software development and data science have already adopted multiple practices addressing the need for project management, collaboration, visualization and strengthening of online communities. As a highly interdisciplinary and growing field, computational biology has yet to acquire a set of universal "best practices". Moreover, as most computational biologists come from diverse backgrounds, they are usually let on their own determining the best ways to conduct their research.
```

Comment:
- [ ] The transition from the previous paragraph to this one feels sudden. We should think of a way of better connecting them, e.g. ending the previous one with an emphasis on using tools we can trust.
- [ ] _"[...] as most computational biologists come from diverse backgrounds, they are usually let on their own determining the best ways to conduct their research"_ don't see how those 2 ideas connect, maybe we should clarify further.

Reply:
Totally agree that the transition is not smooth. I will revisit it.
Regarding the second point, what I wanted to say is that there is no formal education as "bioinformatician" or "computational biologist" that includes a standard set of practices taught at University, for example. Therefore, computational biologist are usually let on their own learning best practices to do research. Would you agree with that?